src/components/ReportTransaction.js

```diff
@@ -23,10 +32,17 @@ const propTypes = {
 
     /** Text label for the reject transaction button */
     rejectButtonLabelText: PropTypes.string.isRequired,
+
+    /* Onyx Props */
+
+    /** List of transactionIDs in process of rejection */
+    /* eslint-disable-next-line react/no-unused-prop-types */
+    rejectInProgress: PropTypes.arrayOf(PropTypes.string),
 };
 
 const defaultProps = {
     canBeRejected: false,
+    rejectInProgress: [],
```

Comment:
NAB; I think we can remove this if we already default it to `[]` in `render()` (line 73). Not super important though, what do you think?

Reply:
I'll have to disable eslint react/require-default-props rule for the prop, which seems alright.

Reply:
If we removed it, rejectedID is passed as undefined for the first time to rejectTransaction.
So, I've added it back.